### ISSUE TITLE
Touch support?

### ISSUE THREAD

**Issue by jaminroe** [igorlino/elevatezoom-plus#87]
Does this library have support for touch? I'm not a developer, so it's hard for me to tell. I don't see anything mentioned on the ReadMe or website, so I'm asking here. Thanks! 😄

**Comment by igorlino** [comment]
Yes there is touch support. (at least from what I can see in the code, have not tried myself.)

If you happen to try it  and give some feedback would be great.
